vivisect/extensions/__init__.py

```diff
@@ -28,19 +28,25 @@ def loadExtensions( vw, vwgui ):
             continue
 
         for fname in os.listdir(dirname):
-
-            if not fname.endswith('.py'):
+            modpath = os.sep.join([dirname, fname])
+            # dig into first level of directories and exec the __init__.py
+            if os.path.isdir(modpath):
+                modpath = os.sep.join([modpath, '__init__.py'])
+                if not os.path.exists(modpath):
+                    continue
+
+            # otherwise, run all the .py files in the VIV_EXT_PATH dir
+            if not modpath.endswith('.py'):
                 continue
 
             # Build code objects from the module files
             mod = imp.new_module('viv_ext')
-            filepath = os.path.join(dirname, fname)
-            filebytes = file( filepath, 'r' ).read()
-            mod.__file__ = filepath
+            filebytes = file( modpath, 'r' ).read()
```

Comment:
Please don't use `file`. I know it works in python 2, but it got removed as a keyword in python 3.

Reply:
lol, that wasn't something i changed.
fixing anyway.